### PR TITLE
Add note about supporting state for WinUI 3

### LINKED ISSUE #9087: Xaml designer for WinIU?
I'm in the process of moving across one of my UWP apps to WinUI and noticed there's no designer? Is that right? Where is the designer for WinUI?

Thanks!

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: eeeb6491-1f2c-73a7-fef2-afb69fd08948
* Version Independent ID: 7e3209b0-a116-3488-a80f-993edf9515a4
* Content: [Create UIs with Visual Studio XAML Designer - Visual Studio (Windows)](https://learn.microsoft.com/en-us/visualstudio/xaml-tools/creating-a-ui-by-using-xaml-designer-in-visual-studio?view=vs-2022)
* Content Source: [docs/xaml-tools/creating-a-ui-by-using-xaml-designer-in-visual-studio.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/main/docs/xaml-tools/creating-a-ui-by-using-xaml-designer-in-visual-studio.md)
* Product: **visual-studio-windows**
* Technology: **vs-xaml-tools**
* GitHub Login: @maddymontaquila
* Microsoft Alias: **maleger**

### PR DESCRIPTION
**Pull request by soroshsabz**
ITNOA

fixes #9087

### CODE REVIEW

**Comment by ShannonLeavitt** [comment]
@maddymontaquila 

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/xaml-tools/creating-a-ui-by-using-xaml-designer-in-visual-studio.md [hunk 1]
```diff
@@ -27,6 +27,9 @@ For advanced users, you can even [customize the XAML Designer](https://github.co
 > [!NOTE]
 > Xamarin.Forms does not support a XAML designer. To view your Xamarin.Forms XAML UIs and edit them while the app is running, use XAML Hot Reload for Xamarin.Forms. For more information, see the [XAML Hot Reload for Xamarin.Forms (Preview)](/xamarin/xamarin-forms/xaml/hot-reload/) page.
 
+> [!NOTE]
+> WinUI 3 / .NET MAUI XAML designer is currently under development and does not support at now, to view your WinUI 3 / .NET MAUI XAML UIs and edit them while the app is running, use XAML Hot Reload for WinUI 3 / .NET MAUI. For more information, see the [XAML Hot Reload](../xaml-tools/xaml-hot-reload.md) page.
```
Comment (ghogen) [inline docs/xaml-tools/creating-a-ui-by-using-xaml-designer-in-visual-studio.md:31]:
```suggestion
> WinUI 3 / .NET MAUI XAML designer is currently under development and is not supported in Visual Studio 2022. To view your WinUI 3 / .NET MAUI XAML UIs and edit them while the app is running, use XAML Hot Reload for WinUI 3 / .NET MAUI. For more information, see the [XAML Hot Reload](../xaml-tools/xaml-hot-reload.md) page.
```

Comment (ghogen) [reply docs/xaml-tools/creating-a-ui-by-using-xaml-designer-in-visual-studio.md:31]: @maddymontaquila Please check my changes and review / approve the PR.

Comment (maddymontaquila) [reply docs/xaml-tools/creating-a-ui-by-using-xaml-designer-in-visual-studio.md:31]: Only note is that it is **not** currently under development - so as long as you can clarify that everything else is good! Thank you!!

**Review by maddymontaquila — CHANGES_REQUESTED** [review]

File: docs/xaml-tools/creating-a-ui-by-using-xaml-designer-in-visual-studio.md (hunk 1 above)
Comment (ghogen) [reply docs/xaml-tools/creating-a-ui-by-using-xaml-designer-in-visual-studio.md:31]: Revising the text to avoid saying it's under development.

**Review by maddymontaquila — APPROVED** [review]

**Pull request merged**